beacon_node/src/config.rs

```diff
@@ -1512,11 +1507,10 @@ where
 
     if values.len() > 1 {
         warn!(
-            log,
-            "Multiple values provided";
-            "info" => "multiple values are deprecated, only the first value will be used",
-            "count" => values.len(),
-            "flag" => flag_name
+            info = "multiple values are deprecated, only the first value will be used",
+            count = values.len(),
+            flag = flag_name,
+            "Multiple values provided"
```

Comment:
it seems a bit off putting that a structured field (in this case `info`) is longer than the log itself. This is my suggestion, but maybe others have differing opinions

```suggestion
            info = "Multiple values provided",
            count = values.len(),
            flag = flag_name,
            "multiple values are deprecated, only the first value will be used"
```